src/sdk/pynni/nni/__main__.py

```diff
@@ -28,6 +28,7 @@
 import importlib
```

Comment:
could you pass pylint for all the modified python files?

Reply:
please ignore this comment, not ready now.

Reply:
OK, thanks